w3-hw1-ThuyDung/cypress/e2e/bookApplication.cy.js

```diff
@@ -0,0 +1,33 @@
+import { loginPage } from "../pages/bookApplication/loginPage";
+import { profilePage } from "../pages/bookApplication/profilePage";
+import { bookPage } from "../pages/bookApplication/bookPage";
+
+describe('Add a book', () =>{
+    beforeEach(() => {
+        cy.fixture("user").as("user");
+        cy.visit(Cypress.env("login"));
+    });
+    it("Verify that user can add a book into their collection when typing book name on search textbox", ()=>{
+        cy.get('@user').then((user) => {
+            loginPage
+                .typeUserName(user.profile.username)
+                .typePassword(user.profile.password)
+                .clickLogin();
+            profilePage
+                .isNavigateToProfile()
+                .isUserNameCorrect(user.profile.username)
```

Comment:
No need to verify here, when the actions are pre-conditions, you can assume that they work correctly. If you verify here it will be duplicate with login cases. Also below, no need to verify at any step you do, E2E is you begin a flow, follow the steps and at the end, what are you expected will be verify